frontend/src/index.pug

```diff
@@ -127,6 +127,14 @@ html
             label
               input(type="checkbox", v-model="filterBreakdown").mui-checkbox.summary-picker__checkbox
               span breakdown by file format
+        #error-message-box(v-if="errorMessages.length")
+          .error-message-box
+            #error-message-box__close-button(onclick="this.parentElement.style.display='none'") &times
```

Comment:
`&` -> `&amp;`

Reply:
I intended to display an `X` and not ampersand.

But do you mean to add the semicolon `;` right after the entity?